mLRS/Common/hal/esp/rx-hal-generic-2400-td-pa-esp32.h

```diff
@@ -106,13 +106,24 @@ IRQHANDLER(void SX2_DIO_EXTI_IRQHandler(void);)
 
 void sx2_init_gpio(void)
 {
+    gpio_init(SX2_CS_IO, IO_MODE_OUTPUT_PP_HIGH);
     gpio_init(SX2_DIO1, IO_MODE_INPUT_ANALOG);
     gpio_init(SX2_BUSY, IO_MODE_INPUT_PU);
     gpio_init(SX2_TX_EN, IO_MODE_OUTPUT_PP_LOW);
     gpio_init(SX2_RX_EN, IO_MODE_OUTPUT_PP_LOW);
     gpio_init(SX2_RESET, IO_MODE_OUTPUT_PP_LOW);
 }
 
+IRAM_ATTR static inline void sx2_select(void)
```

Comment:
I like that better, moving it to the hal
would you mind calling it spi2_select/deselct() nevertheless
should make the sx driver canonical